app/institutions/dashboard/route.ts

```diff
@@ -27,9 +28,19 @@ export default class InstitutionsDashboardRoute extends Route {
                     include: ['summary_metrics'],
                 },
             });
-            const departmentMetrics = yield institution.loadAll('departmentMetrics');
+            const departmentMetrics = yield institution.queryHasMany('departmentMetrics');
             const summaryMetrics = yield institution.summaryMetrics;
-            return { institution, departmentMetrics, summaryMetrics };
+            const userMetricInfo: QueryHasManyResult<never> = yield institution.queryHasMany(
+                'userMetrics',
+                { size: 0 },
```

Comment:
what does `size: 0` do here?

Reply:
We use `userMetricsInfo` only to get the total count, so this just makes sure we aren't loading anything extraneous.